Touch up flows section of docs (#14162)
PrefectHQ/prefect: docs/3.0rc/develop/write-flows/index.mdx

## Parameters

As with any Python function, you can pass arguments to a flow including both positional and keyword arguments.

> These arguments defined on your flow function are called [parameters](/3.0rc/develop/write-flows/#parameters) and are stored by the Prefect orchestration engine on the flow run object.

Reviewer: ```suggestion
These arguments defined on your flow function are called [parameters](/3.0rc/develop/write-flows/#parameters).
They are stored by the Prefect orchestration engine on the flow run object.
```

However you run the flow, the Prefect API monitors the flow run, capturing flow run state for observability.

<Tip>
**Logging**

> Prefect enables you to log a variety of useful information about your flow and task runs. You can capture information 

Reviewer: ```suggestion
Prefect enables you to log a variety of useful information about your flow and task runs. 
```

> about your workflows for purposes such as monitoring, troubleshooting, and auditing.

Reviewer: ```suggestion
You can capture information about your workflows for purposes such as monitoring, troubleshooting, and auditing.
```

Check out [Logging](/3.0rc/develop/observe-workflows/logging) for more information.
</Tip>

When you run a flow that contains tasks or additional flows, Prefect tracks the relationship of each child run to the parent flow run.

![Prefect UI](/3.0rc/img/ui/timeline-flows.png)

<Tip>
**Retries**

Unexpected errors may occur. For example the GitHub API may be temporarily unavailable or rate limited.

> Check out [Retries](/3.0rc/develop/recover-from-failure) to help make flows more resilient.

Reviewer: ```suggestion
Check out [Retries](/3.0rc/develop/recover-from-failure) to learn how to make your flows more resilient.
```
